### DISCUSSION TITLE
推荐一款开源的低代码平台vlife

### DISCUSSION THREAD

**Discussion by wwwlike213** [taowen/awesome-lowcode#122] [General]
全栈开源的模型驱动的低代码平台。
- [vlife 低代码平台](http://vlife.cc/)
  - 基于 react+springboot 技术栈，编写 Javabean 就能完成业务功能开发，前后端开源的全栈低代码平台
  - [产品介绍](https://github.com/wwwlike/vlife)
  - [平台演示](http://admin.vlife.cc/)
  - [产品视频](https://www.bilibili.com/video/BV1sT411c71v/?vd_source=4c025d49e1ac4adb74b6dd2a39ce185e&t=119.6)

**Comment by LowCoderGitHub** [comment]
搞了个低代码行业的交流平台，大佬们有没有兴趣来新平台玩玩
https://www.lowcoder.com.cn/
